src/main/java/io/specto/hoverfly/junit/core/RemoteHoverflyConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.specto.hoverfly.junit.core;
+
+public interface RemoteHoverflyConfig extends HoverflyConfig {
+
+
+    RemoteHoverflyConfig withHttps();
```

Comment:
Yes, it will get SSL cert from environment variable, but it is not implemented yet.